devito/passes/iet/openmp.py

```diff
@@ -122,13 +123,14 @@ def _make_construct(cls, parallel=False, **kwargs):
 
     @classmethod
     def _make_clauses(cls, ncollapse=None, chunk_size=None, nthreads=None,
-                      reduction=None, **kwargs):
+                      reduction=None, schedule=None, **kwargs):
```

Comment:
could almost make it an `opt` option like openmp/..... if people want to put their own ?

Reply:
I prefer them to play with JIT BACKDOOR and if they find out in some specific cases that a certain schedule is better than others, then I'd prefer to implement the logic/heuristic in the compiler directly. Long story short: I don't think it's a knob we should be exposing, because our experiments show that basically we always do the right thing